README.md

```diff
@@ -10,19 +11,32 @@ exist long enough to be scraped, they can instead push their metrics
 to a Pushgateway. The Pushgateway then exposes these metrics to
 Prometheus.
 
-The Pushgateway is explicitly not an aggregator, but rather a metrics
-cache. It does not have a statsd-like semantics. The metrics pushed
-are exactly the same as you would present for scraping in a
-permanently running program.
+## Non-goals
+
+The Pushgateway is explicitly not an _aggregator or distributed counter_ but
+rather a metrics cache. It does not have a statsd-like semantics. The metrics
+pushed are exactly the same as you would present for scraping in a permanently
+running program.
 
 For machine-level metrics, the
 [textfile](https://github.com/prometheus/node_exporter/blob/master/README.md#textfile-collector)
-collector of the Node exporter is usually more appropriate. The Pushgateway is best
-used for service-level metrics.
+collector of the Node exporter is usually more appropriate. The Pushgateway is
+best used for service-level metrics.
```

Comment:
/s/best used/intended for/ ? Purely an opinion, nothing wrong with what is currently written.